include/libtorrent/disk_buffer_holder.hpp

```diff
@@ -86,6 +86,7 @@ namespace libtorrent {
 		char* release() noexcept;
 
 		// return a pointer to the held buffer
+		char* data() const noexcept { return m_buf; }
 		char* get() const noexcept { return m_buf; }
```

Comment:
any reason to keep `get()`?

Reply:
the disk cache still uses ``get()`` and I thought it might not be worth changing all of its uses